0000-template.md

```diff
@@ -1,52 +1,51 @@
----
-rfc pr: [#xxx](https://github.com/aws/aws-cdk-rfcs/pull/xxx) <-- fill this after you've already created the PR
-tracking issue: https://github.com/aws/aws-cdk-rfcs/issues/yyy
----
+# [#NNN](http://github.com/aws/aws-cdk-rfcs/issues/NNN) [TITLE]
```

Comment:
minor: Not a fan of this format. I prefer a separate line above or below this heaeder linking to the tracking issue.

Reply:
Fixed